src/lib/storage/fixed_size_attribute_vector.hpp

```diff
@@ -0,0 +1,38 @@
+#pragma once
+
+#include <vector>
+#include "storage/base_attribute_vector.hpp"
+#include "utils/assert.hpp"
+
+namespace opossum {
+
+template <typename T>
+class FixedSizeAttributeVector : public BaseAttributeVector {
+ public:
+  explicit FixedSizeAttributeVector(std::vector<T> value_ids) : _value_ids{value_ids} {}
+
+  // returns the value id at a given position
+  virtual ValueID get(const size_t index) const {
+    DebugAssert(index < size(), "Index out of bounds");
```

Comment:
You're using a DebugAssert here and a `Assert` below, I think that should be consistent.